src/java/src/main/java/example/demo/Response.java

```diff
@@ -1,6 +1,3 @@
-// Copyright 2018 SMF Authors
-//
-
```

Comment:
there is a tool that autoformats all the source code and epxects clang-format 0.6.0 (avail on fedora 29)

do you mind running it via `./tools/build.sh -f `

Feel free to submit a PR for the fedora:29 docker image.

Reply:
let me know if you have any issues here. Happy to help.